SocialCareCaseViewerApi/V1/Boundary/Requests/CreateAllocationRequest.cs

```diff
@@ -14,5 +14,11 @@ public class CreateAllocationRequest
 
         [Required]
         public string AllocatedBy { get; set; }
```

Comment:
I don't think this is necessary anymore, but we can sort it out when we update this to support teams as well. We get the allocated worker id anyway, so just need the CreatedBy as additional value